core/sequencer/sequencer.go

```diff
@@ -68,7 +68,7 @@ func init() {
 
 // Sequencer processes mutations and sends them to the trillian map.
 type Sequencer struct {
-	admin     adminstorage.Storage
+	admin     domainstorage.Storage
```

Comment:
Does it still make sense for this field to be named `admin`?

Reply:
Fixed.